source/jormungandr/jormungandr/street_network/tests/handimap_test.py

```diff
@@ -260,7 +260,7 @@ def get_language_handimap_func_language_invalid_test():
     instance = MagicMock()
     handimap = Handimap(instance=instance, service_url=fake_service_url)
     language = handimap._get_language("toto")
-    assert language == "fr-FR"
+    assert language == "en-EN"
```

Comment:
`en-EN` doesn't exist. See [here](https://en.wikipedia.org/wiki/Language_localisation#Language_tags_and_codes) for an exhaustive list of existing variants.

Reply:
Since list of languages managed by handimap is not modified, i didn't even test if transformation of english to en-EN is valid or not

Reply:
I verified in the deo handimap:
https://dev.keolis-rennes.handimap.fr/docs/#model-RouteRequest
language	Languagesstring
default: en-EN
example: fr-FR

Reply:
Thank you very much for showing up and posting this :smile: